WordPress/src/main/java/org/wordpress/android/ui/accounts/SignupEpilogueActivity.java

```diff
@@ -51,7 +61,12 @@ protected void addSignupEpilogueFragment(String name, String email, String photo
 
     @Override
     public void onContinue() {
-        setResult(RESULT_OK);
+        if (mSiteStore.hasSite()) {
+            setResult(RESULT_OK);
+        } else if (AppPrefs.shouldShowPostSignupInterstitial()) {
```

Comment:
Here, I think we shouldn't check for the preference value and always show the interstitial. I can't think of any use case where we shouldn't nudge new users to create a new site.

Reply:
Can we guarantee a new user will never have a site? If so, I think we can remove the `SiteStore.hasSites()` check. Wasn't sure about this at first, because the user can log in with a self-hosted site (but apparently not _sign up_ with one).

The preference check is needed here to comply with the requirement of only showing the interstitial **once per app install**. Maybe @SylvesterWilmott can give us a bit of context about this.

Reply:
>Can we guarantee a new user will never have a site? If so, I think we can remove the SiteStore.hasSites() check. Wasn't sure about this at first, because the user can log in with a self-hosted site (but apparently not sign up with one).

I can't think of any scenarios where a new user will already have a site but is there a downside to performing the `hasSites` check on sign up?

>The preference check is needed here to comply with the requirement of only showing the interstitial once per app install. Maybe @SylvesterWilmott can give us a bit of context about this.

I don't have any strong opinions about this. We can show the interstitial per account created.

Reply:
> I can't think of any scenarios where a new user will already have a site but is there a downside to performing the `hasSites` check on sign up?

I just confirmed _there is_ a way for a new user to already have a site: 
- Log in with a self-hosted site address.
- Go to the _Me_ tab and tap **Log in to WordPress.com**.
- You'll be redirected to the [login prologue screen](https://user-images.githubusercontent.com/830056/71200202-6093b680-2276-11ea-9836-0af22937c0f8.png), where you'll be able to enter the signup flow and create a new account normally.

Do we still want to always show the interstitial in this case or can I leave the `hasSites` check?

Reply:
Yes and the API might change, user creation might end up with a site created, so better to keep the `hasSites()` check anyway.